### PR TITLE
fix flake8 and dead code

### PR DESCRIPTION
**Pull request by esc**
As title.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2019, Anaconda, Inc.
+Copyright (c) 2019-2020, Anaconda, Inc.
```
Comment (stuartarchibald) [inline LICENSE]: Unsure it's necessary to keep moving this? Perhaps @seibert will recall.

File: README.rst [hunk 2]
```diff
@@ -52,5 +47,5 @@ In order to add a new target, you need to:
 License
 =======
 
-Copyright 2019 Anaconda Inc. Licensed under the terms of the BSD two-clause
-license. See the file ``LICENSE`` for details.
+Copyright 2019-2020 Anaconda Inc. Licensed under the terms of the BSD
```
Comment (stuartarchibald) [inline README.rst]: Same comment RE updates as above.

**Review by stuartarchibald — COMMENTED** [review]
Thanks for the patch. Couple of minor things to resolve else looks good.

File: LICENSE (hunk 1 above)
Comment (seibert) [reply LICENSE]: although counter-intuitive, we were told by our lawyer some years ago not to use date ranges for copyrights.  They basically said the important thing was that the initial date was given, and the current copyright holder.

Comment (esc) [reply LICENSE]: Thanks, I'll revert this change then.

**Comment by esc** [comment]
Changes have been made, I believe it is now ready?

**Comment by esc** [comment]
Everything has been addressed here and is up to data against today's master.

**Review by stuartarchibald — APPROVED** [review]
Thanks for the fixes.

**Pull request merged**